src/components/CardMenus/Tabs.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { Paper, Tab, Tabs } from '@material-ui/core';
+
+const MainTab = () => {
+  const [value, setValue] = React.useState(2);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+  return (
+    <>
+      <Paper square>
+        <Tabs
+          aria-label="disabled tabs example"
+          indicatorColor="primary"
+          onChange={handleChange}
+          textColor="primary"
+          value={value}
+        >
+          <Tab label="Active" />
```

Comment:
What that same label "Active" n times means ... are u just render Active for all ?? If yes, then please use some appropriate name ..